README.md

```diff
@@ -273,6 +314,15 @@ environments:
       publishing:
         - 80
 ```
+#### AWS Command Line Access
+
+Follow the guide at Configuring the AWS CLI to use AWS Single Sign-On to set up CLI access with SSO. The details needed are:
+```yaml
+  SSO Start URL: https://ons.awsapps.com/start
+  SSO Region: eu-west-2
+```
+
+Each environment you have access to will need to be configured. Make sure you name each profile the same as the environment (dp-sandbox, dp-staging, dp-prod, dp-ci) so that it works seamlessly with other DP tooling (dp-cli, terraform). A sample config is included at the end of this guide as a reference.
```

Comment:
I can't see a sample config at the end of the guide. I think if we have a guide to configuring the AWS CLI somewhere, just linking to that should be enough.